the_mines/scrape_html.py

```diff
@@ -0,0 +1,92 @@
+from bs4 import BeautifulSoup
+from argparse import ArgumentParser
+
+
+def fussballdatenpunktde_matchday_results(site_html):
```

Comment:
It may be a good idea to move any `fussballdaten` specific code into a `fussballdaten/` directory.  So if we add any other websites in the future we can isolate the different scape files.  This is something we probably don't have to worry about just yet.

Reply:
Yes, I agree